File: bank-service/src/main/java/rs/edu/raf/model/ExchangeRate.java
package rs.edu.raf.model;

import jakarta.persistence.*;
import lombok.*;

import java.math.BigDecimal;

@NoArgsConstructor
@AllArgsConstructor
@Getter
@Setter
@Entity
public class ExchangeRate {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;
    private String currencyCode;
    private BigDecimal rate;
}

File: bank-service-kotlin/src/main/kotlin/org/example/bankservicekotlin/model/ExchangeRate.kt
package org.example.bankservicekotlin.model

import jakarta.persistence.Entity
import jakarta.persistence.GeneratedValue
import jakarta.persistence.GenerationType
import jakarta.persistence.Id
import java.math.BigDecimal

@Entity
class ExchangeRate(
    @Id @GeneratedValue(strategy = GenerationType.IDENTITY)
    var id: Long?,
    val currencyCode: String,
    val rate: BigDecimal) {
}